pkg/genericapiserver/config.go

```diff
@@ -190,6 +190,8 @@ type SecureServingInfo struct {
 	ServerCert GeneratableKeyCert
 	// SNICerts are named CertKeys for serving secure traffic with SNI support.
 	SNICerts []NamedCertKey
+	// ServerCA is the certificate bundle needed to verify the ServerCert
+	ServerCA string
```

Comment:
it's slightly odd for the server to need this... I think the cert file referenced in GeneratableKeyCert or NamedCertKey should already contain the chain